docs/installation.rst

```diff
@@ -44,13 +44,13 @@ build the latest release, you can run::
 Singularity YODA style using ///repronim/containers
 ===================================================
 
-ReproNim project provides `///repronim/containers <http://datasets.datalad.org/?dir=/repronim/containers>`_
+The ReproNim project provides `///repronim/containers <http://datasets.datalad.org/?dir=/repronim/containers>`_
 (git clone present also on `GitHub <https://github.com/ReproNim/containers>`__) `DataLad
 <https://datalad.org>`_ dataset with Singularity containers for many popular neuroimaging tools, e.g. all BIDS-Apps.
 It also contains converted from Docker singularity images for stock heudiconv images (as `nipy-heudiconv
 <https://github.com/ReproNim/containers/tree/master/images/nipy>`__) and reproin images (as `repronim-reproin
 <https://github.com/ReproNim/containers/tree/master/images/repronim>`__). Please see `"A typical workflow"
```

Comment:
The first two sentences of this paragraph are very confusing. What are the intended ideas?